### DISCUSSION TITLE
Is https://homelabber.org using the scripts with permission?

### DISCUSSION THREAD

**Discussion by GTRasH** [tteck/Proxmox#3583] [General]
I can’t see any mentioning of tteck at all but texts and installation instruction from the site are a 1:1 copy.

see https://homelabber.org/t/trilium-proxmox-lxc/232

**Comment by tteck** [comment]
No! Just another copy-cat!!

**Reply by GTRasH** [reply]
They really do not add anything of value

**Reply by tteck** [reply]
Definitely one that I would not trust! I'm not even willing to sign up to express my displeasure.

**Reply by tteck** [reply]
But, we can email info@homelabber.org and ask WTF

**Comment by Mati-l33t** [comment]
Been using without problems, website is a collection of usefull stuff for homelabbers, Proxmox and stuff. Also there is a Thank you and have been for as long ive been there.
![Screenshot_20240825_014842_Chrome](https://github.com/user-attachments/assets/1ae46b0c-a651-44ed-8de5-1bff28708cf3)

**Reply by tteck** [reply]
Recently added?

**Reply by Mati-l33t** [reply]
Date says January 28

**Reply by tteck** [reply]
This site is copying all of my work word for word and passing it off as their original content without any credit. This is unacceptable.

![Screenshot 2024-08-25 6 11 16 AM](https://github.com/user-attachments/assets/83dbea75-5c33-4029-a1b8-66b6d0c467c6)

**Reply by tteck** [reply]
This kind of blatant copying, without giving proper credit, is the reason why developers lose motivation to continue their hard work.
